lib/views/home/home_view.dart

```diff
@@ -18,9 +18,12 @@ class HomeView extends StatefulWidget {
 class _HomeViewState extends State<HomeView> {
   int _currentIndex = 0;
 
+  double amountSat = 0;
+
   @override
   void initState() {
     super.initState();
+    getamountMsat();
     _currentIndex = 1;
   }
```

Comment:
```suggestion
  void initState() async {
    super.initState();
    await getamountMsat();
    _currentIndex = 1;
  }
```

We need to await this right? or I'm missing the key idea here?

Reply:
We cannot make a initstate function async!